source/data/tier1data/clients/mango_portal.rst

```diff
@@ -0,0 +1,179 @@
+Clicking on the name of a subcollection shows you its contents under the "Content" tab: both subcollections and data objects.
+If you click on a data object instead, you go to its page, which does not have such a "Content" tab.
+
+Above the name of the current collection or data object, a breadcrumb menu allows you to go back to a collection on top of it. 
+Next to the name, clicking on the pencil icon allows you to rename the active collection or data object.
+
+The page of a collection also includes buttons to create a new collection and upload files,
+which are only available if you have the right :ref:`permissions<collaboration>`.
+Also depending on your permissions you may see a dropdown menu of bulk actions, allowing you to move, copy or delete one or more
+collections or data objects at a time.
+To do so, click on the checkbox next to the names of the collections or data objects, select your desired action and click on "apply".
+When copying or moving, a list of collections will appear where you can browse to select a destination.
+Notice that only data objects, not collections, can be copied.
+
+Data objects do not have a "Content" tab, as they cannot contain any objects. 
+Instead, they have a "System properties" tab, which contains some basic information about the object.
+They also provide a "Preview" tab in which, depending on the file type, you may be able to see the contents of your data object.
+
+Next to these specific tabs, both collections and data objects have a :ref:`"Metadata" <edit-metadata>` and :ref:`"Permissions" <edit-permissions>` tab
+for inspection and management of metadata and permissions respectively.
+
+******************************
+Uploading and downloading data
+******************************
+
+Clicking on 'Upload files...' opens a white box, where you can put one or multiple files to be uploaded to the current collection:
+
+- By dragging files from your local pc into the white box.
+- By clicking inside the white box, which opens your file explorer, where you can select your files. 
+
+.. figure:: ../images/mango_portal/mango_portal_upload.png
+  :width: 500
+  :alt: Uploading files via the ManGO portal
+
+If you made a mistake, you can click on 'Remove file' under the file in question. 
+When you are ready, click on 'Start uploading files' to upload your selection. 
+
+To download a data object, from the page of that object you click on the download-icon next to its name.
+Alternatively, you can download it from the page of its parent collection by clicking on the download
+icon next at the end of its row.
+
+.. figure:: ../images/mango_portal/mango_portal_download.png
+  :width: 800
+  :alt: Downloading files via the ManGO portal
+
+Uploads and downloads via the ManGO portal are limited to 1GB and 20GB per file respectively.
+You can upload multiple files at the same time, but you can only download data objects individually.
```

Comment:
Bulk download is already available, This part needs to be updated

Reply:
Applied